static/app/views/performance/newTraceDetails/traceRow/traceBar.tsx

```diff
@@ -157,9 +157,7 @@ interface TraceBarProps {
 }
 
 export function TraceBar(props: TraceBarProps) {
-  const duration = props.node_space
-    ? formatTraceDuration(props.node_space[1], isTransactionNode(props.node) ? 0 : 2)
-    : null;
+  const duration = props.node_space ? formatTraceDuration(props.node_space[1], 2) : null;
```

Comment:
We need to preserve txn check here as the precision differs between spans and transaction. That said, we need to only round to ms if the duration of the transaction is less than a second